Cargo.toml

```diff
@@ -12,6 +12,5 @@ readme = "README.md"
 
 [dependencies]
 isatty = "0.1.1"
-
-[target."cfg(unix)".dependencies]
+duct = "0.10"
```

Comment:
It looks like some dependency of duct fails to build on Rust 1.17. Please bump the version in .travis.yml to the smallest one that duct supports.